hcloud/routes_test.go

```diff
@@ -83,6 +83,15 @@ func TestRoutes_CreateRoute(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Unexpected error: %v", err)
 	}
+
+	err = routes.CreateRoute(context.TODO(), "my-cluster", "routeFail", &cloudprovider.Route{
+		Name:            "route",
+		TargetNode:      "node15",
+		DestinationCIDR: "172.16.0.0/16",
+	})
+	if err == nil {
+		t.Fatalf("Expected an error because DestinationCIDR is not within the cluster CIDR, but received nil instead")
+	}
```

Comment:
Could you test the error string too, please? Maybe with `require.ErrorRegexp` or `require.Error` and `require.Regexp`